server/container_create_unsupported.go

```diff
@@ -0,0 +1,25 @@
+// +build !linux
+
+package server
+
+import (
+	"fmt"
+
+	"github.com/kubernetes-incubator/cri-o/lib/sandbox"
+	"github.com/kubernetes-incubator/cri-o/oci"
+	"github.com/opencontainers/runtime-tools/generate"
+	"golang.org/x/net/context"
+	pb "k8s.io/kubernetes/pkg/kubelet/apis/cri/runtime/v1alpha2"
+)
+
+func findCgroupMountpoint(name string) error {
+	return fmt.Errorf("no cgroups on this platform")
```

Comment:
This sort of thing means we need to rework our internal interfaces to get platform-agnostic helpers.  But that sort of cleanup can happen in follow-up work; this PR is already pretty big ;).

Reply:
ya